### PR TITLE
✨  Add ability to pass argument to init scripts to pick json format

### PR DESCRIPTION
**Pull request by svpernova09**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-cp -i src/stubs/Homestead.yaml Homestead.yaml
+if [[ -n "$1" ]]; then
```
Comment (andrewhood125) [inline init.sh]: How bout `[ "$1" == "--json" ]`?

**Comment by andrewhood125** [comment]
https://github.com/laravel/homestead/issues/486

File: init.sh (hunk 1 above)
Comment (svpernova09) [inline init.sh]: I don't care *what* the argument is. Essential if any argument exists, use JSON

Comment (andrewhood125) [inline init.sh]: As someone not familiar with this whole conversation inspecting the code though, its easier to get context. Oh I pass in `--json` and get a json stub. It's a minor, but anything we can do to make the code easier to understand.

**Comment by svpernova09** [comment]
Tagging @DojoGeekRA, this is the PR I mentioned in the email

**Pull request merged**